aiida/cmdline/commands/cmd_devel.py

```diff
@@ -38,13 +38,8 @@ def get_valid_test_paths():
     db_prefix_raw = 'db'
     db_prefix = db_prefix_raw + '.'
     base_test_modules = [
-        'aiida.cmdline.utils',
-        'aiida.cmdline.params.types',
-        'aiida.cmdline.params.options',
-        'aiida.common',
-        'aiida.schedulers',
-        'aiida.transports',
-        'aiida.tools.dbimporters.plugins',
+        'aiida.cmdline.utils', 'aiida.cmdline.params.types', 'aiida.cmdline.params.options', 'aiida.common',
+        'aiida.schedulers', 'aiida.transports', 'aiida.tools.dbimporters.plugins', 'aiida.tools.visualization'
```

Comment:
I think all the tests are actually in the module `aiida.backends` correct? And I don't see any in `aiida.tools.visualization`, so it is not necessary to add it here

Reply:
fixed in 6e599e9